app/controllers/sessions_controller.rb

```diff
@@ -28,7 +28,9 @@ def destroy
   # POST /users/login
   def create
     user = User.find_by(email: session_params[:email])
-    if user.try(:authenticate, session_params[:password])
+    if user && !user.greenlight_account? && session_params[:password]
```

Comment:
Do we really need the veryfy that password is included?
```
if user && !user.greenlight_account? && session_params[:password]
```
shouldn't be something like this
```
if user && !user.greenlight_account?
```
Password will be always there as the form passes it either empty or with a value